src/documentation/fares_internal_format.md

```diff
@@ -0,0 +1,32 @@
+# Representing fares in Navitia Transit Model
+## Overview
+This document describes how fare structures are modeled in Navitia Transit Model. The internal model is composed of the followning objects:
+- Ticket
+- OD Rules
+
+## Model details
+### Ticket
+This object contains the necessary information to specify a ticket.
+
+Property | Type | Required | Description
+--- | --- | --- | ---
+id | String | Yes | Ticket identifier
+input_data_format | Enum | Yes | Source of the fare data, fixed value `nl_syntus_fares`.
+start_date | Date | Yes | Start date for the validity period of the ticket price.
+end_date | Date | Yes | End date for the validity period of the ticket price. This date is included in the validity period interval.
+currency_type | String | Yes | The currency used to pay the ticket. The [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217#Active_codes) currency codes are used.
+price | Float | Yes | The total price of the ticket in the currency specified by `currency_type`, including any additional fee, if any (e.g. boarding fee).
+validity_duration | Integer | No | Validity duration of the ticket in seconds.
+transfers | Integer | No | Number of transfers allowed for the ticket.
```

Comment:
you should specify the behaviour when `validity_duration` and `transfers` are not specified (no duration limit and no/unlimited transfers ?)